tests/test_prettytable.py

```diff
@@ -344,6 +344,11 @@ def init_db(db_cursor):
 class TestBasic:
     """Some very basic tests."""
 
+    def test_table_rows(self, city_data_prettytable: PrettyTable) -> None:
+        rows = city_data_prettytable.rows[:]
```

Comment:
And can we skip making a copy here?
```suggestion
        rows = city_data_prettytable.rows
```